core/web_admin.py

```
@@ -0,0 +1,1040 @@
+)
+from telethon.sessions import StringSession
+
+from astrbot.api import logger
+
+from .client import TelegramClientWrapper
+
+PLUGIN_NAME = "astrbot_plugin_telegram_forwarder"
+
+DEFAULT_WEB_CONFIG = {
+    "enabled": True,
+    "host": "0.0.0.0",
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)